Packs/DevSecOps/Integrations/GitLab/GitLab.yml

```diff
@@ -641,7 +653,401 @@ script:
+      description: The merge request merge commit SHA.
+      type: String
+    - contextPath: GitLab.MergeRequest.squash_commit_sha
+      description: The merge request squash commit SHA.
+      type: String
+  - arguments:
+      - default: false
+        description: Project ID from which to retrieve the Merge Requests.
+        isArray: false
+        name: project_id
+        required: true
+        secret: false
+      - default: false
+        description: Merge Request IID.
+        isArray: false
+        name: merge_request_iid
+        required: true
+        secret: false
+    description: Get a Merge Request.
+    name: gitlab-get-merge-request
+    outputs:
+      - contextPath: GitLab.MergeRequest.id
+        description: The merge request ID.
+        type: Number
+      - contextPath: GitLab.MergeRequest.iid
+        description: The merge request IID.
+        type: Number
+      - contextPath: GitLab.MergeRequest.project_id
+        description: The project ID of the merge request.
+        type: Number
+      - contextPath: GitLab.MergeRequest.title
+        description: The merge request title.
+        type: String
+      - contextPath: GitLab.MergeRequest.description
+        description: The merge request description
+        type: String
+      - contextPath: GitLab.MergeRequest.state
+        description: The merge request state.
+        type: String
+      - contextPath: GitLab.MergeRequest.created_at
+        description: The time the merge request was created.
+        type: Date
+      - contextPath: GitLab.MergeRequest.updated_at
+        description: The time the merge request was updated.
+        type: Date
+      - contextPath: GitLab.MergeRequest.merged_at
+        description: The time the merge request was merged.
+        type: Date
+      - contextPath: GitLab.MergeRequest.closed_by
+        description: The user who closed the merge request.
+        type: String
+      - contextPath: GitLab.MergeRequest.closed_at
+        description: The time the merge request was closed.
+        type: Date
+      - contextPath: GitLab.MergeRequest.target_branch
+        description: The merge request target branch.
+        type: String
+      - contextPath: GitLab.MergeRequest.source_branch
+        description: The merge request source branch.
+        type: String
+      - contextPath: GitLab.MergeRequest.assignee
+        description: The merge request assignee.
+        type: String
+      - contextPath: GitLab.MergeRequest.sha
+        description: The merge request commit SHA.
+        type: String
+      - contextPath: GitLab.MergeRequest.merge_commit_sha
+        description: The merge request merge commit SHA.
+        type: String
+      - contextPath: GitLab.MergeRequest.squash_commit_sha
+        description: The merge request squash commit SHA.
+        type: String
+  - arguments:
+    - name: labels
+      description: Comma seperated values of labels to search issues by.
+    - name: in
```

Comment:
```suggestion
    - name: scope
```

Reply:
I think it is better to leave it as `in` as it is in the API.

Reply:
I guessed that this is the reason you chose this name but I don't think it is very clear. You can obviously set it to whatever you want later in the code.